Dockerfile

```diff
@@ -1,13 +1,13 @@
 
-# docker build -t mgrast/auth-server .
+# docker build -t mgrast/authserver .
 
-# docker rm auth-server ; docker run -d --name auth-server -p 7000:80  mgrast/auth-server
+# docker rm authserver ; docker run -d --name authserver -p 7000:80  mgrast/authserver
```

Comment:
Did you check if the name change has no side effects?

Reply:
Yes, I checked.